src/test/java/niko/command/AddCommandTest.java

```diff
@@ -0,0 +1,34 @@
+package niko.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import niko.main.Storage;
+import niko.main.Ui;
+import niko.task.Task;
+import niko.task.TaskList;
+import niko.task.Todo;
+
+
```

Comment:
Might be good to cleanup areas with excessive number of empty lines. Noticed some of these in other areas of the code as well.